### ISSUE TITLE
✨ Admin pools table - new column - Publishing group

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#6090]
## ✨ Feature

Include a new column for displaying the publishing group a pool belongs to on the Pools page

## 🕵️ Details

- Ideally this will also be included in the filters on this table to allow easier finding.

## 📸 Screenshot

![image](https://user-images.githubusercontent.com/16977254/228033649-18c580fc-f249-409f-ad15-d3b5538fcee5.png)

## 🙋‍♀️ Proposed Implementation

(optional) Do you have a proposed implementation?

## 🌎 Localization

(optional) Provide any new copy along with translations available.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] New column for publishing group
- [ ] Visible by default
- [ ] sortable (filtering out of scope)

## 🛑 Blockers

Blocked by {issue numbers}
